WordPress/Classes/ViewRelated/Gutenberg/GutenbergSuggestionsViewController.swift

```diff
@@ -23,8 +22,7 @@ public class GutenbergMentionsViewController: UIViewController {
 
     public lazy var searchView: UITextField = {
         let textField = UITextField(frame: CGRect.zero)
-        textField.placeholder = NSLocalizedString("Search users...", comment: "Placeholder message when showing mentions search field")
```

Comment:
This placeholder is not needed because the text field always has an `@` character plus any search query the user has typed. If the user backspaces to remove the `@` character, this view controller is dismissed so the placeholder text is never shown.